src/mmw/js/src/core/sidebarToggleControl.js

```diff
@@ -3,19 +3,25 @@
 var L = require('leaflet'),
     $ = require('jquery'),
     Marionette = require('../../shim/backbone.marionette'),
+    MapModel = require('./models'),
     sidebarToggleControlButtonTmpl = require('./templates/sidebarToggleControlButton.html');
 
 module.exports = L.Control.extend({
     options: {
-        position: 'bottomright'
+        position: 'bottomleft'
+    },
+
+    initialize: function(options) {
+        this.mapModel = options.model;
     },
 
     onAdd: function() {
-        return new SidebarToggleControlButton({}).render().el;
+        return new SidebarToggleControlButton({ model: this.mapModel }).render().el;
     }
 });
 
 var SidebarToggleControlButton = Marionette.ItemView.extend({
+    model: MapModel,
```

Comment:
Just so that you're aware, putting `model` inside a Marionette `extend` doesn't actually _do_ anything other than perhaps indicate what kind of model this class should receive when instantiated. And if documentation is all we want, we can do that with a comment instead of a `require`.